Examples/Swift/LineStyleLayerExample.swift

```diff
@@ -69,7 +84,7 @@ class LineStyleLayerExample_Swift: UIViewController, MGLMapViewDelegate {
         // Line gap width represents the space before the outline begins, so should match the main line’s line width exactly.
         casingLayer.lineGapWidth = layer.lineWidth
         // Stroke color slightly darker than the line color.
-        casingLayer.lineColor = NSExpression(forConstantValue: UIColor(red: 41/255, green:145/255, blue:171/255, alpha:1))
+        casingLayer.lineColor = NSExpression(forConstantValue: UIColor(red: 41/255, green: 145/255, blue: 171/255, alpha: 1))
         // Use `NSExpression` to gradually increase the stroke width between zoom levels 14 and 18.
         casingLayer.lineWidth = NSExpression(format: "mgl_interpolate:withCurveType:parameters:stops:($zoomLevel, 'linear', nil, %@)",
                                              [14: 1, 18: 4])
```

Comment:
Nit: would you mind removing this line break and merging it into a single line? Feels weird, dangling out there.